source/Calamari/Scripts/Octopus.Features.IISWebSite_BeforePostDeploy.ps1

```diff
@@ -81,32 +81,79 @@ $supportsSNI = $iisVersion -ge 8
 
 $wsbindings = new-object System.Collections.ArrayList
 
-# Each binding string consists of a protocol/binding information (IP, port, hostname)/SSL thumbprint/enabled/sslFlags
-# Binding strings are pipe (|) separated to allow multiple to be specified
-$bindingString.Split("|") | foreach-object {
-    $bindingParts = $_.split("/")
-	$skip = $false
-	if ($bindingParts.Length -ge 4) {
-		if (![String]::IsNullOrEmpty($bindingParts[3]) -and [Bool]::Parse($bindingParts[3]) -eq $false) {
-			$skip = $true
+Write-Output $bindingString
+
+if($bindingString.StartsWith("[{")) {
+	
+	if(Get-Command ConvertFrom-Json -errorAction SilentlyContinue){
+		$bindingArray = (ConvertFrom-Json $bindingString)
+	} else {
+		add-type -assembly system.web.extensions
```

Comment:
`ConvertFrom-Json` is only in PS 3 but would be nice to not have to rely on `system.web.extensions` so fallback to this only if in PS 2 or less.